docs/integration.md

```diff
@@ -0,0 +1,129 @@
+﻿# Integration with Hedera <-> EVM-chain bridge
+The Bridge provides users with functionality to transfer HBAR or HTS tokens from Hedera to EVM-based chain or Wrapped HBAR and Wrapped Tokens from EVM-based chain to Hedera. The Bridge is operated by registered validators that provide signatures for every requested transfer. The transfer is processed when the majority of validators verify the transfer (supermajority).
+
+## 1. Hedera -> EVM-chain
```

Comment:
I think that this document should be focused on what exactly should be done by the integrator step-by-step.
So it will be better if we do not have 1.1 How it works, but rather ->
- Step 1. Initiate transfer of HBAR/HTS
- Step 2. Wait for confirmation
- Step 3. Claim Asset on EVM chain

The goal would be for someone to read the document and know what he has to do in order to integrate